docs/development/extensions-contrib/kafka-emitter.md

```diff
@@ -36,20 +36,28 @@ to monitor the status of your Druid cluster with this extension.
+| property                                           | description                                                                                                                                                                                                                                                             | required? | default               |
+|----------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|-----------------------|
+| `druid.emitter.kafka.bootstrap.servers`            | Comma-separated Kafka broker. (`[hostname:port],[hostname:port]...`)                                                                                                                                                                                                    | yes       | none                  |
+| `druid.emitter.kafka.event.types`                  | Comma-separated event types. <br/>Choices: alerts, metrics, requests, segmentMetadata                                                                                                                                                                                   | no        | ["metrics", "alerts"] |
+| `druid.emitter.kafka.metric.topic`                 | Kafka topic name for emitter's target to emit service metric. If `event.types` contains `metrics`, this field cannot be left empty                                                                                                                                      | no        | none                  |
+| `druid.emitter.kafka.alert.topic`                  | Kafka topic name for emitter's target to emit alert. If `event.types` contains `alerts`, this field cannot be left empty                                                                                                                                                | no        | none                  |
+| `druid.emitter.kafka.request.topic`                | Kafka topic name for emitter's target to emit request logs. If `event.types` contains `requests`, this field cannot be left empty                                                                                                                                       | no        | none                  |
+| `druid.emitter.kafka.segmentMetadata.topic`        | Kafka topic name for emitter's target to emit segments related metadata. If `event.types` contains `segmentMetadata`, this field cannot be left empty. <br/>If `segmentMetadata.topic.format` is set to `protobuf`, then the topic name will be suffixed by `-protobuf` | no        | none                  |
```

Comment:
it's not good practice to include a format name in a topic. Format and schema are usually defined out of band in a schema-registry so that they can evolve without impacting the topic name. We should use the same default regardless of the format.

Reply:
Also, this makes for a poor configuration UX, since someone reading the config will assume that the topic that is configured is the one that it writes to. Magically adding some kind of suffix to it based on some other config makes this very confusing and error-prone